fairseq/options.py

```diff
@@ -506,6 +506,10 @@ def add_generation_args(parser):
                        help='maximum iterations for iterative refinement.')
     group.add_argument('--iter-decode-force-max-iter', action='store_true',
                        help='if set, run exact the maximum number of iterations without early stop')
+    group.add_argument('--iter-decode-with-beam', default=1, type=int, metavar='N',
+                       help='if > 1, model will generate translations varying by the lengths.')
+    group.add_argument('--iter-decode-with-external-reranker', action='store_true',
+                       help='if set, the last checkpoint are assumed to be a reranker to rescore the translations'),
```

Comment:
I think we can actually move all of these into the task and then when decoding you can do:
```generate.py --task transformer_lev (...)```. Lemme check with @kahne and see if we can clean this up.